### ISSUE TITLE
can't install on k8s

### ISSUE THREAD

**Issue by IMOKURI** [ansible/awx#4737]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY

Display following error when clean install on k8s.

```json
TASK [kubernetes : Migrate database] ******************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "kubectl -n awx exec ansible-tower-management --  bash -c \"awx-manage migrate --noinput\"\n", "delta": "0:00:05.399162", "end": "2019-09-13 15:01:31.015263", "msg": "non-zero return code", "rc": 1, "start": "2019-09-13 15:01:25.616101", "stderr": "Traceback (most recent call last):\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 217, in ensure_connection\n    self.connect()\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 195, in connect\n    self.connection = self.get_new_connection(conn_params)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/postgresql/base.py\", line 178, in get_new_connection\n    connection = Database.connect(**conn_params)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/psycopg2/__init__.py\", line 130, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: FATAL:  password authentication failed for user \"awx\"\n\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/usr/bin/awx-manage\", line 11, in <module>\n    load_entry_point('awx==7.0.0.0', 'console_scripts', 'awx-manage')()\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/awx/__init__.py\", line 142, in manage\n    execute_from_command_line(sys.argv)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/__init__.py\", line 381, in execute_from_command_line\n    utility.execute()\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/__init__.py\", line 375, in execute\n    self.fetch_command(subcommand).run_from_argv(self.argv)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/base.py\", line 323, in run_from_argv\n    self.execute(*args, **cmd_options)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/base.py\", line 364, in execute\n    output = self.handle(*args, **options)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/base.py\", line 83, in wrapped\n    res = handle_func(*args, **kwargs)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/commands/migrate.py\", line 87, in handle\n    executor = MigrationExecutor(connection, self.migration_progress_callback)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/executor.py\", line 18, in __init__\n    self.loader = MigrationLoader(self.connection)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/loader.py\", line 49, in __init__\n    self.build_graph()\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/loader.py\", line 212, in build_graph\n    self.applied_migrations = recorder.applied_migrations()\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/recorder.py\", line 73, in applied_migrations\n    if self.has_table():\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/recorder.py\", line 56, in has_table\n    return self.Migration._meta.db_table in self.connection.introspection.table_names(self.connection.cursor())\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 256, in cursor\n    return self._cursor()\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 233, in _cursor\n    self.ensure_connection()\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 217, in ensure_connection\n    self.connect()\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/utils.py\", line 89, in __exit__\n    raise dj_exc_value.with_traceback(traceback) from exc_value\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 217, in ensure_connection\n    self.connect()\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 195, in connect\n    self.connection = self.get_new_connection(conn_params)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/postgresql/base.py\", line 178, in get_new_connection\n    connection = Database.connect(**conn_params)\n  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/psycopg2/__init__.py\", line 130, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\ndjango.db.utils.OperationalError: FATAL:  password authentication failed for user \"awx\"\n\ncommand terminated with exit code 1", "stderr_lines": ["Traceback (most recent call last):", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 217, in ensure_connection", "    self.connect()", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 195, in connect", "    self.connection = self.get_new_connection(conn_params)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/postgresql/base.py\", line 178, in get_new_connection", "    connection = Database.connect(**conn_params)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/psycopg2/__init__.py\", line 130, in connect", "    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)", "psycopg2.OperationalError: FATAL:  password authentication failed for user \"awx\"", "", "", "The above exception was the direct cause of the following exception:", "", "Traceback (most recent call last):", "  File \"/usr/bin/awx-manage\", line 11, in <module>", "    load_entry_point('awx==7.0.0.0', 'console_scripts', 'awx-manage')()", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/awx/__init__.py\", line 142, in manage", "    execute_from_command_line(sys.argv)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/__init__.py\", line 381, in execute_from_command_line", "    utility.execute()", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/__init__.py\", line 375, in execute", "    self.fetch_command(subcommand).run_from_argv(self.argv)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/base.py\", line 323, in run_from_argv", "    self.execute(*args, **cmd_options)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/base.py\", line 364, in execute", "    output = self.handle(*args, **options)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/base.py\", line 83, in wrapped", "    res = handle_func(*args, **kwargs)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/commands/migrate.py\", line 87, in handle", "    executor = MigrationExecutor(connection, self.migration_progress_callback)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/executor.py\", line 18, in __init__", "    self.loader = MigrationLoader(self.connection)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/loader.py\", line 49, in __init__", "    self.build_graph()", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/loader.py\", line 212, in build_graph", "    self.applied_migrations = recorder.applied_migrations()", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/recorder.py\", line 73, in applied_migrations", "    if self.has_table():", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/migrations/recorder.py\", line 56, in has_table", "    return self.Migration._meta.db_table in self.connection.introspection.table_names(self.connection.cursor())", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 256, in cursor", "    return self._cursor()", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 233, in _cursor", "    self.ensure_connection()", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 217, in ensure_connection", "    self.connect()", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/utils.py\", line 89, in __exit__", "    raise dj_exc_value.with_traceback(traceback) from exc_value", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 217, in ensure_connection", "    self.connect()", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/base/base.py\", line 195, in connect", "    self.connection = self.get_new_connection(conn_params)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/db/backends/postgresql/base.py\", line 178, in get_new_connection", "    connection = Database.connect(**conn_params)", "  File \"/var/lib/awx/venv/awx/lib64/python3.6/site-packages/psycopg2/__init__.py\", line 130, in connect", "    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)", "django.db.utils.OperationalError: FATAL:  password authentication failed for user \"awx\"", "", "command terminated with exit code 1"], "stdout": "", "stdout_lines": []}

PLAY RECAP ********************************************************************************************************************************************************************************************************
localhost                  : ok=24   changed=10   unreachable=0    failed=1    skipped=47   rescued=0    ignored=3   

```

I can install when I've modified inventory as follows.

```diff
 # This will create or update a default admin (superuser) account in AWX, if not provided
 # then these default values are used
-admin_user=admin
+admin_user=awx
 admin_password=password
```

##### ENVIRONMENT
* AWX version: 7.0.0
* AWX install method: k8s
* Ansible version:  2.8.4
* Operating System: CentOS 7.6.1810
* Web Browser:

##### STEPS TO REPRODUCE

- setup k8s
- update inventory file
- exec `ansible-playbook -i inventory install.yml`

##### EXPECTED RESULTS

successfully installed

##### ACTUAL RESULTS

installer failed

**Comment by ryanpetrello** [comment]
@shanemcd @rooftopcellist is this related to https://github.com/ansible/awx/pull/4734 ?

**Comment by IMOKURI** [comment]
It seems fixed on 8.0.0. Thanks!

**Comment by silveiralexf** [comment]
Had exact same issue on 9.0.1 -- after changing `admin_user` to `awx` the install successfully finishes.
